hypnotoad/cases/tokamak.py

```diff
@@ -1452,17 +1513,16 @@ def Bt_axis(self):
         return self.fpol(self.psi_axis) / self.o_point.R
 
 
-def read_geqdsk(filehandle, options={}, **kwargs):
+def read_geqdsk(filehandle, settings={}, nonorthogonal_settings={}, make_regions=True):
```

Comment:
Thanks for pointing this out - it is much better to avoid potentially persistent default arguments! I think I'd rather make `OptionsFactory.create(None)` a valid thing to do, then the empty dicts wouldn't be needed at all.